tests/backstop_data/engine_scripts/puppet/activity-feed-panel/activity-detail.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const Utility = require('./../utility.js');
+
+module.exports = async (page, scenario, vp) => {
+  const utility = new Utility(page, scenario, vp);
+
+  await utility.waitForAngular();
+  await utility.waitForLoadingComplete();
+  await page.click('.civicase__activity-feed__list .civicase__activity-card:not(.civicase__activity-card--draft)');
```

Comment:
Not sure why are we clicking on `:not(.civicase__activity-card--draft)`?

Reply:
We are clicking on `.civicase__activity-feed__list .civicase__activity-card` which is . a normal activity card. The `:not(.civicase__activity-card--draft)` part is so we don't click an activity card for a draft. Not sure if this is relevant, but it does work either way.

Reply:
Can we please make sure, and if `:not` is really required, otherwise lets remove it. 
Lets not put unnecessary logics if if works without the same.

Reply:
It does work the same, drafts can be clicked as normal and I don't see a difference between a draft and a normal activity:

### Draft
![screen shot 2019-02-26 at 7 20 25 am](https://user-images.githubusercontent.com/1642119/53409369-24196600-3997-11e9-8014-c1b5baf5cf5a.png)

### Normal
![screen shot 2019-02-26 at 7 15 40 am](https://user-images.githubusercontent.com/1642119/53409386-2bd90a80-3997-11e9-91bc-eabdcaef461e.png)